### PR TITLE
Add channel-and-reseller page

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add `/partners/channel-and-reseller` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/partners/channel-and-reseller
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](https://docs.google.com/document/d/1rn2f4wL8wxnILlnmfchkNQV2RzsYUMcRaCfP-d6V_5Q)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2567

### CODE REVIEW

**Comment by waynecrosby** [comment]
@solazio Thanks for that, could you update the page to match the design below please. I'm not 100% sure about the panel in the header as it doesn't make sense to me, but I think Thibaut may have added it?
Also, the logos in the 'logo cloud' will need to be replaced with ID = Reseller ones.
Reseller

Link for laptop image:
Mid-size: https://assets.ubuntu.com/v1/a5b7c95b-laptop-find-partner-sml.png
Small: https://assets.ubuntu.com/v1/da448671-laptop-find-partner-sml2.png

File: templates/partners/channel-and-reseller.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    }
+  }
+</style>
+<section class="p-strip--light is-shallow" style="margin-top: -1px;">
+  <div class="row">
+    <div class="col-8">
+      <p>Interested in becoming an Ubuntu partner? Talk to us today to learn more about our programmes.</p>
+    </div>
+    <div class="col-4">
+      <a href="/partners/become-a-partner" class="p-button--positive">Get in touch</a>
```
Comment (deadlight) [inline templates/partners/channel-and-reseller.html:58]: It's weird that there are two links to the same form in the header.

Comment (solazio) [reply templates/partners/channel-and-reseller.html:57]: I removed one link

File: templates/partners/channel-and-reseller.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  <div class="u-fixed-width u-align--center">
+    <h4 class="p-muted-heading u-no-max-width">Meet some of our channel partners</h4>
+  </div>
+  <div class="u-fixed-width">
+    <ul class="p-inline-images u-no-margin--bottom">
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/cb3fcdfb-lenovo.svg" alt="Lenovo">
+      </li>
+      <li class="p-inline-images__item" style="display: inline; padding-top: 0.5rem;">
+        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/d258d23b-2018-logo-dell.svg" alt="Dell">
```
Comment (deadlight) [inline templates/partners/channel-and-reseller.html:72]: The dell logo is displaying weirdly for me:
image

I know that I've mentioned this on other pages but these logo clouds shouldn't be hard-coded. We want non-webteam people to be able to change them in the CMS

Comment (solazio) [reply templates/partners/channel-and-reseller.html:72]: I will need to update all the partners pages to use the CMS data. I have a [separate issue](https://github.com/canonical-web-and-design/canonical.com/issues/214) for that.

Comment (deadlight) [reply templates/partners/channel-and-reseller.html:72]: Ah, no problem

**Pull request merged**